demo/admin/package.json

```diff
@@ -32,6 +32,7 @@
         "@emotion/react": "^11.9.3",
         "@emotion/styled": "^11.9.3",
         "@fontsource/roboto": "^4.5.5",
+        "@graphql-codegen/near-operation-file-preset": "^2.5.0",
```

Comment:
All the other graphql-codegen packages are in dev dependencies, please move it there